core/server/config/index.js

```diff
@@ -212,7 +212,8 @@ ConfigManager.prototype.set = function (config) {
         },
         deprecatedItems: ['updateCheck', 'mail.fromaddress'],
         // create a hash for cache busting assets
-        assetHash: assetHash
+        assetHash: assetHash,
+        preloadHeaders: false
```

Comment:
Setting this value here overrides any value provided by the user in config.js. It would need to be set something like this to enable it to be overridden:

`preloadHeaders: this._config.preloadHeaders || false`